app/main/views/briefs.py

```diff
@@ -472,5 +483,4 @@ def list_brief_services(brief_id):
     )
 
     services = services.filter(Service.supplier_id == supplier.supplier_id)
-
-    return jsonify(services=[service.serialize() for service in services])
+    return list_result_response("services", services), 200
```

Comment:
We could do `from app.main.views.services import RESOURCE_NAME as SERVICE_RESOURCE_NAME` here?